sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java

```diff
@@ -323,6 +329,116 @@ void testAssociateNewLocationForHandleSameStateTwice() {
         assertTrue(testRunObserver.isInvalid(), "Test run should have been invalid.");
     }
 
+    @Test
+    @DisplayName("testMetricStatusManipulationMSRMTActivationStateSHTDNGood: Set ActivationState "
+            + "of all MSRMT-Metrics to SHTDN.")
+    void testMetricStatusManipulationMSRMTActivationStateSHTDNGood() throws PreconditionException {
+        final ComponentActivation startActivationState = ComponentActivation.ON;
```

Comment:
You could introduce a test setup method to reduce duplicate code, similar to the other tests

Reply:
Unfortunately, the setup code initializes a lot of variables and contains a lot of references to these variables. When some part of it is extracted into a setup method, then these references all break. The only way I could reduce the duplicated code is by extracting the whole thing into a method with parameters that allow configuring it to run the different test cases. However, that would be significantly less readable than the current solution, I think.

Reply:
Currently, one can at least use a diff tool to see what has changed between the Good case and the Bad cases...

Reply:
You could do a general setup and only adjust the specific things in the unittest itself. Then the readability wouldn't suffer either. Similar to [here](https://github.com/Draegerwerk/SDCcc/blob/83c6ea6dd39c3cd601ea4a14b6b05ce76fb69435/sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java#L204)

Reply:
Done.